platform/darwin/src/MGLStyleAttributeValue_Private.hpp

```diff
@@ -17,4 +19,7 @@
 - (mbgl::style::PropertyValue<std::vector<float> >)mbgl_numberArrayPropertyValue;
 - (mbgl::style::PropertyValue<uint8_t>)mbgl_enumPropertyValue;
 
+// Convert mbgl types to darwin types
+- (id <MGLStyleAttributeValue>)mbgl_colorPropertyValueWith:(mbgl::style::PropertyValue<mbgl::Color>)color;
```

Comment:
Oh, right, `-mbgl_colorWithPropertyValueColor:` makes sense. I got confused about the return type because of the existing name. 😄 Also, if you ever find the lack of generics to be limiting, we can also break out a standalone, templated C++ function and call that instead of the Objective-C category method.